tools/bashstyle

```diff
@@ -6,7 +6,7 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright 2020 Joyent, Inc.
```

Comment:
If this file was just copied from elsewhere, it probably shouldn't have a copyright update.  But that begs the question: why copy it from `deps/eng/tools`?  Why not just use the copy that is there?

Reply:
Yeah, I just removed it. Not really used.